### ISSUE TITLE
Add more buildings to the New Zealand dataset

### ISSUE THREAD

**Issue by simonschaufi** [facebook/Rapid#982]
Some cities in New Zealand are growing rapidly and therefor some cities lack some buildings. Would it be possible to request just certain areas to scan again and suggest more buildings?

It would be awesome if it could be possible right in the editor to select an area and request a scan.

Example in Cambridge: https://rapideditor.org/rapid#background=LINZ_NZ_Aerial_Imagery&datasets=ef9136b079e24bbfb432be9bbdd246ff&disable_features=boundaries&map=16.92/-37.92282/175.48239

**Comment by Bonkles** [comment]
Great idea and thanks for the request @simonschaufi. Right now I don't think we have a very good story for 'continually updating' datasets from ESRI. Till now they've been mostly datasets that represent a snapshot in time. The features get added to OSM, and once done, the feature set is retired or taken offline. 

I'll be attending ESRI UC next week and will bring up the idea of 'tasking' and/or continually refreshing a dataset such as this with our partners there.

**Comment by simonschaufi** [comment]
Great!

**Comment by Bonkles** [comment]
chatted with ESRI about this today, expect a response from them soon in this space. :)

**Comment by dkensok** [comment]
Yes, we talked and we are planning to prepare an update of the LINZ buildings shortly (next couple weeks) and will update the layer that is hosted in ArcGIS Online and accessible in Rapid.  We'll post an update here when that update is done!
